lib/srv/termmanager_test.go

```diff
@@ -89,41 +88,43 @@ func TestBufferedKept(t *testing.T) {
 	require.Equal(t, m.buffer, kept)
 }
 
-type funcReader func(p []byte) (n int, err error)
-
-func (f funcReader) Read(p []byte) (n int, err error) {
-	return f(p)
-}
-
 func TestNoReadWhenOff(t *testing.T) {
 	m := NewTermManager()
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*15)
```

Comment:
nit: perhaps we could go with something more conservative than fifteen seconds or we could make this `t.Parallel()`. Having read the implementation of the underlying code, I know that this test will be extremely quick, but if I hadn't and was glancing at this test, I might wonder why a test that could run for 5-10 seconds isn't in parallel.

Reply:
Marked as parallel.